addons/payment_stripe/models/account_payment_method.py

```diff
@@ -10,5 +10,5 @@ class AccountPaymentMethod(models.Model):
     @api.model
     def _get_payment_method_information(self):
         res = super()._get_payment_method_information()
-        res['stripe'] = {'mode': 'unique', 'domain': [('type', '=', 'bank')]}
+        res['stripe'] = {'mode': 'multi', 'domain': [('type', '=', 'bank')]}
```

Comment:
I have no idea if that is correct since it's `account`-related (maybe @vin-odoo could have a look?) but, if so, it should be done for every payment provider, not just Stripe.